lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -55,6 +59,7 @@ def compute(self, student, course_structure, scores_client, submissions_scores):
             # self.scores may hold outdated data, force it to refresh on next access
             lazy.invalidate(self, 'scores')
 
+        log.info("accessed lazy property scores with value: {0}".format(unicode(self.scores)))
```

Comment:
I think we should definitely at least include the "computed" statements. I wouldn't mind keeping all of them, with the "accessed" statements at the info level, too. And then we should test these lazy properties as well.